scenario/utils.ts

```diff
@@ -319,11 +319,7 @@ export async function fastGovernanceExecute(
   signatures: string[],
   calldatas: string[]
 ) {
-  const governor = await dm.contract('governor');
-
-  if (!governor) {
-    throw new Error("cannot create governance proposal without governor");
-  }
+  const governor = await dm.getContractOrThrow('governor');
```

Comment:
Can this just be the behavior or `contract()`?

Reply:
not without further refactoring: `clone`, `deploy` and `existing` all use `this.contract` in a way that would break if it threw for missing contracts. they check for an existing contract and take different actions if it's not there